codenav/src/utils.c

```diff
@@ -19,7 +19,7 @@
  *  along with this program; if not, see <http://www.gnu.org/licenses/>.
  */
 
-#include "utils.h"
+#include "geanyplugin.h"
```

Comment:
I got this warning, and the change fixed it for me. I didn't realize there is also a local utils.h
  CC       utils.lo
utils.c: In Funktion »compare_strings«:
utils.c:92:2: Warnung: Implizite Deklaration der Funktion »utils_str_equal« [-Wimplicit-function-declaration]
  return (gint)(!utils_str_equal(a, b));

It's bad that that codenav has a header called util.h, it'll always clash. The convention should have been to #include <geany/geanyplugin.h> but that's a lot of changes.